concepts/conditionals-if/about.md

```diff
@@ -13,12 +13,14 @@ if value == "val" {
 In scenarios involving more than one case many `if` statements can be chained together using the `else if` and `else` statements.
 
 ```go
-if value == "val" {
-    // conditional code
-} else if value == "val2" {
-    // conditional code
+var number int
+
+if number > 0 {
+    return "positive"
+} else if number < 0 {
+    return "negative"
 } else {
-    // default code
+    return "zero"
 }
```

Comment:
In this example all the else statements are obsolete because of the returns.  
There are a couple of options to fix this that come to mind:

**Option 1**
Use fmt.Println instead of return. Drawback is the student might not know this function but it is pretty self explanatory.

**Option 2**
Show a less concrete example. Drawback is that it is less concrete. 
```go
if number > 0 {
    // Code that should be executed for positive numbers
} else if number < 0 {
    // Code that ....
} else {
    // Code that ...
}
```

**Option 3**
Change the example so returning is not the best option anymore. Make the example more complex.
```go
var number int
result := "This number is "

if number > 0 {
    result += "positive"
} else if number < 0 {
    result +="negative"
} else {
    result += "zero"
}
```

@jmrunkle Any preference?

Reply:
I trust your judgement and it looks like it has been merged now. :D